oxd-common/src/main/java/org/xdi/oxd/common/ErrorResponseCode.java

```diff
@@ -37,6 +38,7 @@
     INVALID_ID_TOKEN_UNKNOWN(500, "invalid_id_token_unknown", "Invalid id_token, validation fail due to exception, please check oxd-server.log for details."),
     INVALID_ACCESS_TOKEN_BAD_HASH(500, "invalid_access_token_bad_hash", "access_token is invalid. Hash of access_token does not match hash from id_token (at_hash)."),
     INVALID_AUTHORIZATION_CODE_BAD_HASH(500, "invalid_authorization_code_bad_hash", "Authorization code is invalid. Hash of authorization code does not match hash from id_token (c_hash)."),
+    INVALID_AUTHORIZATION_CODE_BAD_CODE(400, "invalid_authorization_code", "Authorization code is invalid. Please use the code from authorization URL."),
```

Comment:
add word about expiration, so developer is aware about expiration. `Authorization code is invalid or otherwise expired`.